// expo-server-sdk/build/ExpoClientValues.js
"use strict";
var _a;
Object.defineProperty(exports, "__esModule", { value: true });
exports.requestRetryMinTimeout = exports.defaultConcurrentRequestLimit = exports.pushNotificationReceiptChunkLimit = exports.pushNotificationChunkLimit = exports.getReceiptsApiUrl = exports.sendApiUrl = void 0;
/**
 * The URLs for the Expo push service endpoints.
 *
 * The EXPO_BASE_URL environment variable is only for internal Expo use
 * when testing the push service locally.
 */
const baseUrl = (_a = process.env.EXPO_BASE_URL) !== null && _a !== void 0 ? _a : 'https://exp.host';
exports.sendApiUrl = `${baseUrl}/--/api/v2/push/send`;
exports.getReceiptsApiUrl = `${baseUrl}/--/api/v2/push/getReceipts`;
/**
 * The max number of push notifications to be sent at once. Since we can't automatically upgrade
 * everyone using this library, we should strongly try not to decrease it.
 */
exports.pushNotificationChunkLimit = 100;
/**
 * The max number of push notification receipts to request at once.
 */
exports.pushNotificationReceiptChunkLimit = 300;
/**
 * The default max number of concurrent HTTP requests to send at once and spread out the load,
 * increasing the reliability of notification delivery.
 */
exports.defaultConcurrentRequestLimit = 6;
/**
 * Minimum timeout in ms for request retries.
 */
exports.requestRetryMinTimeout = 1000;
